cmd/testtingzone/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+)
+
+func main() {
+	// This is just for me to test the scripts
```

Comment:
What you can do instead of this is create a file called `pkg/scrapers/jobcenter_scraper_test.go` (*_test.go is a special file name) to create unit test for the function.

```go
package jobcenter_scraper

import ....

func TestGetJobcenterJobId(t *testing.T) {
    jobId := getJobcenterJobId(".....")
    if jobId != "1234" {
        t.Errorf("Expected %s but got %s", "expected", "got")
    }
}
```

Then you can run `go test` in that directory to run the test, or you can run `go test ./...` at the repository root to run all tests.